### PR TITLE
:memo: update version of c and c++ in readme

### PR DESCRIPTION
**Pull request by liam-teale**
simple PR, see title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Primary maintenance of PROS is done by students at Purdue University through [Pu
 
 All PROS development is open sourced and available for the community to inspect. We believe that sharing source improves PROS through community contributions and allows students to learn by example.
 
-PROS is built using the GCC toolchain and standard C/C++ practices (C11 & C++17 w/GNU extensions) to make the learning curve small. Structures, pointers, dynamic memory allocation, and function pointers are all available. Additionally, code is run on bare metal, allowing you to take full advantage of the microcontroller's power.
+PROS is built using the GCC toolchain and standard C/C++ practices (C2x & C++23 w/GNU extensions) to make the learning curve small. Structures, pointers, dynamic memory allocation, and function pointers are all available. Additionally, code is run on bare metal, allowing you to take full advantage of the microcontroller's power.
```
Comment (Rocky14683) [inline README.md]: it should be C23

**Review by Rocky14683 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (liam-teale) [reply README.md]: isn't the compiler flag C2X?

**Comment by liam-teale** [comment]
@Rocky14683 ready for another review

**Review by Rocky14683 — APPROVED** [review]
LGTM

**Pull request merged**